### PR TITLE
Adding Render pages

### PR DESCRIPTION
**Pull request by PopoTheGamer**
Adding Render pages, for introducing some Black MIDI rendering options.

### CODE REVIEW

**Comment by GreenWeegeeCantCode** [comment]
this definitely wont be merged because this has some spelling errors, also UMP and zenith shouldnt be in one doc, they should be seperate guides and there shouldnt be just 1 guide with 2 rendering programs, before making a pr please check for any errors and if it contains multiple programs make them seperate guides.

File: docs/Render/images/sample [hunk 1]
```diff
@@ -0,0 +1 @@
+nothing here
```
Comment (6gh) [inline docs/Render/images/sample]: You should delete this file

File: docs/Render/index.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# Renderring Black MIDI
```
Comment (6gh) [inline docs/Render/index.md]: Rendering*

File: docs/Render/index.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+## Zenith
+
+??? info "What is Zenith?"
+
+    Zenith is an open-source Black MIDI renderer made by Arduano. It is a great option for the starter to render Black MIDI easily.
+
+Zenith also includes note count rendering, which can be customized easily and has more infos
+
+Zenith has 2 versions which you can choose to start with:
```
Comment (6gh) [inline docs/Render/index.md:9]: Maybe explain a little of what the modded version does?

File: docs/Render/index.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+-   [The original Zenith](https://github.com/arduano/Zenith-MIDI), made by Arduano
+-   [Modded version](https://github.com/null7323/Zenith-MIDI-Modded), made by null7323
+
+![Original Zenith Application when started](images/ZenithPreview.png)
```
Comment (6gh) [inline docs/Render/index.md]: I think we should move this right under the "# Zenith" header

File: docs/Render/index.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+## Ultralight MIDI Player
+
+??? info "What is Ultralight MIDI Player?"
+
+    Ultralight MIDI Player (abbreviated as UMP) is a Java-based Black MIDI renderer developed by PipiraMine. Not only for rendering, it is also possible to play Black MIDI there. It is known to the community because of how fast it goes when rendering and customizing the skins.
+
+UMP has both release and beta versions. The release version has no bugs meanwhile beta version has early features included. 
+
+![UMP playing a MIDI](images\UMPPreview.png)
```
Comment (6gh) [inline docs/Render/index.md]: Same thing with the Zenith image, maybe move this to right under the header

**Review by 6gh — CHANGES_REQUESTED** [review]
Hey! Thanks for taking the time to submit a pull request with your suggestions! While this isn't in the format of the other guides, nor does it explain how to install and set up these programs, it did give me an idea to make a "Start" section for absolute complete beginners who don't even know where to start. I will probably use this to make a "Rendering MIDIs" page in there.

With this new idea in mind, I think your changes are great for what I explained above. Please use my comments to fix it up and I will do the rest in creating that new, "Start", section. Thank you again for contributing!

**Comment by 6gh** [comment]
> this definitely wont be merged because this has some spelling errors, also UMP and zenith shouldnt be in one doc, they should be seperate guides and there shouldnt be just 1 guide with 2 rendering programs, before making a pr please check for any errors and if it contains multiple programs make them seperate guides.

You are not wrong, I would prefer for UMP and Zenith to have their own section. Although this PR did give me a new idea (explained a bit in the comment I left) so I can still probably use this PR.

In short, I'll make a new section called "Start" to have pages such as "How to make Black MIDI", "How to play Black MIDI", and "How to render Black MIDI" (this is where this PR comes in), where in these pages they then link to the actual guides on how to setup a variety of programs. This way newcomers who know nothing of the process can more quickly learn which programs do what and only read the pages they want to.

Let me know what you think of this idea, I'm open to suggestions as always!

> PS: Please try not to be so harsh on new contributors. Maybe it wasn't your intention, but your comment here and on the closed PR gives off a passive aggressive tone and overall not welcoming. It's partly my fault for procrastinating on finishing the Contributing guide for this project, which I will finish up when I get on a PC. Thank you for looking out for this project though!

**Comment by PopoTheGamer** [comment]
Somehow I've included both Zenith and UMP into a single page because the page was meant to be an introduction to both of the programs and will let newcomers choose one out of it to start rendering their first Black MIDI video. That's why I didn't include the "how to install and set up" thing because I think it would be good to be done on separate pages.

**Review by 6gh — APPROVED** [review]
Looks good! I will make a "Start" section using this. Thank you for your contribution!

**Pull request merged**